store/displayPreferences.ts

```diff
@@ -1,116 +1,104 @@
-import Vue from 'vue';
 import { DisplayPreferencesDto } from '@jellyfin/client-axios';
-import { MutationTree, ActionTree, GetterTree } from 'vuex';
-import nuxtConfig from '~/nuxt.config';
+import { MutationTree, ActionTree } from 'vuex';
+import { merge, toString as toStr } from 'lodash';
```

Comment:
We use it in a lot of other places so using individual modules is unmanageable. We're aware about this, treeshaking is just not a priority right now. We will revise this at some point, don't worry 👍.

Reply:
@Codex- I know that lodash has a really bad reputation (I don't understand exactly why though). The thing that we're trying to with thic client is to avoid what happened to old web: an unmaintained mess of custom code.

So we're following this approach for this: if there's a library that does it for us, it's better than do it ourselves.

The pages you linked have all good examples of  how to write those methods with pure Javasscript but, let's be honest: isn't clear and concise what you see here stated as ``merge`` instead of a 20 line loop?